pkg/server/plugin/upstreamauthority/awssecret/awssecret_test.go

```diff
@@ -130,7 +142,11 @@ func (as *Suite) testCSRTTL(plugin upstreamauthority.Plugin, preferredTTL int32,
 	csr, _, err := util.NewCSRTemplate(validSpiffeID)
 	as.Require().NoError(err)
 
-	resp, err := plugin.MintX509CA(ctx, &upstreamauthority.MintX509CARequest{Csr: csr, PreferredTtl: preferredTTL})
+	stream, err := plugin.MintX509CA(ctx, &upstreamauthority.MintX509CARequest{Csr: csr, PreferredTtl: preferredTTL})
+	as.Require().NoError(err)
+	as.Require().NotNil(stream)
+
+	resp, err := stream.Recv()
```

Comment:
We should add an additional call to Recv() somewhere after here that we assert returns io.EOF.